shared/search/index.js

```diff
@@ -50,7 +52,7 @@ export default connector.connect(
      showUserGroup,
      selectedUsers,
      onRemoveUserFromGroup: user => { dispatch(removeUserFromGroup(user)) },
-     onClickUserInGroup: user => { dispatch(selectUserForInfo(user)) },
+     onClickUserInGroup: user => { dispatch(isMobile ? routeAppend({path: 'profile', username: user.username}) : selectUserForInfo(user)) },
```

Comment:
i think this is fine for now. we'll see what patterns emerge as we have more divergent mobile/desktop stuff. other option would be just one action creator, but in the creator itself it does the checking